files/ODA/ODA_IfcTunnel_Sprint_1_2_Georef_A.ifc

```diff
@@ -0,0 +1,52 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [notYetAssigned]'),'2;1');
+FILE_NAME('e:/ODA_IfcTunnel_Sprint_1_2.ifc','2023-03-29T23:26:29',(''),('Open Design Alliance'),'ODA SDAI 24.4','','810681');
+FILE_SCHEMA(('IFC4X4'));
+ENDSEC;
+
+DATA;
+#1=IFCPERSON('Vov-A6 Lab Tech',$,'Vovchik',$,$,$,$,$);
+#2=IFCORGANIZATION($,'Open Design Alliance',$,$,$);
+#3=IFCPERSONANDORGANIZATION(#1,#2,$);
+#4=IFCORGANIZATION($,'Company 1','File produced in ODA IFC SDK ExIfcTutorials example',$,$);
+#5=IFCAPPLICATION(#4,'0.1 pre alpha','ExIfcTutorials module for ODA IFC SDK','ExIfcTutorials');
+#6=IFCOWNERHISTORY(#3,#5,$,.ADDED.,1680121588,$,$,0);
+#7=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#8=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#9=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#10=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#11=IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.017453292519943295),#10);
+#12=IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#13=IFCCONVERSIONBASEDUNIT(#12,.PLANEANGLEUNIT.,'degree',#11);
+#14=IFCSIUNIT(*,.MASSUNIT.,.KILO.,.GRAM.);
+#15=IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#16=IFCUNITASSIGNMENT((#7,#8,#9,#13,#14,#15));
+#17=IFCPROJECT('0DauD5KiTAwxOrEudwUKb3',#6,'Ifc Tunnel Deployment',$,$,$,$,(#22,#24),#16);
+#18=IFCCARTESIANPOINT((0.,0.));
+#19=IFCCARTESIANPOINT((0.,0.,0.));
+#20=IFCDIRECTION((0.,0.,1.));
+#21=IFCAXIS2PLACEMENT2D(#18,$);
+#22=IFCGEOMETRICREPRESENTATIONCONTEXT('2D','Plan',2,$,#21,$);
+#23=IFCAXIS2PLACEMENT3D(#19,$,$);
+#24=IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,$,#23,$);
+#25=IFCCLASSIFICATION('Classification Source',$,$,'Classification',$,$,$);
+#26=IFCRELASSOCIATESCLASSIFICATION('1bqcC80$H3P9hmOuRnuYPp',$,$,$,(#17),#25);
+#27=IFCGROUP('2mOAo$rdr86xAMpd4N3X$A',#6,'Just a group',$,$);
+#28=IFCACTORROLE(.USERDEFINED.,'Ifc Tunnel Deployer',$);
+#29=IFCPERSON($,$,'Name of person',$,$,$,(#28),$);
+#30=IFCACTOR('1ArzXT56j1Ow2yxo_OWTUw',#6,'An Ifc Tunnel Actor',$,$,#29);
+#31=IFCRELDECLARES('27HenZXYD8IgOJiJwZSwIY',#6,'Declarations for IfcProject',$,#17,(#27,#30));
+#32=IFCRELAGGREGATES('3hAXu2uxLCp91f9ZSmHKaU',$,$,$,#17,(#33));
+#33=IFCSITE('2Qfe7Ym$z2QecS3_GSjfKh',#6,'Site',$,$,$,$,$,$,$,$,$,$,$);
+#34=IFCRELAGGREGATES('0CCSRWAOLFher0cy3bUw1S',$,$,$,#33,(#35));
+#35=IFCTUNNEL('2g1yk9ccL0J8YLVS85IZQ7',#6,'Tunnel',$,$,$,$,$,$,.ACCESSTUNNEL.);
+#36=IFCPROJECTEDCRS('EPSG:4326',$,'World Geodetic System (WSG84)',$,$,$,$);
+#37=IFCMAPCONVERSION(#24,#36,10000.,15000.,0.,1.,0.,1.);
+#38=IFCWELLKNOWNTEXT('POINT(100 500)',#36);
```

Comment:
The first attribute is incorrect according to the [documentation](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/ifcwellknowntext.htm):

> Text string according to ISO/IEC 13249. If CoordinateReferenceSystem is populated, then the representation is a coordinate reference system according to ISO 19162. 

It uses ISO 13249 instead of ISO 19162, while it's second attribute is populated.

This comment applies to all three files.